src/Sulu/Bundle/WebsiteBundle/DependencyInjection/SuluWebsiteExtension.php

```diff
@@ -128,6 +128,10 @@ public function load(array $configs, ContainerBuilder $container)
             // add alias for default controller
             $container->setAlias(DefaultController::class, 'sulu_website.default_controller')
                 ->setPublic(true);
+
+            if ($container->hasExtension('twig') && class_exists(ExceptionController::class)) {
```

Comment:
`hasExtension` returns false in my case... Might that be because we are relying on the order of the bundles here? Do we need this check at all? The `ExceptionController` is only available if twig is installed, isn't it? AFAIK the only edge case would be if somebody installed twig, but did not register it in the kernel, right?